File: Codeforces/Kotlin/src/main/kotlin/TheNewYearMeetingFriends.kt
package problems

import kotlin.math.abs

/* Problem Link: https://codeforces.com/problemset/problem/723/A */

fun main() {
    val distances = readln().split(" ").map { it.toInt() }.sorted()
    print(abs(distances[0] - distances[1]) + abs(distances[2] - distances[1]))
}

File: Codeforces/C++/problemset/The New Year Meeting Friends.cpp
// Problem Link: https://codeforces.com/problemset/problem/723/A

#include <iostream>
#include <algorithm>
using namespace std;
typedef long long ll;
int main()
{
    ll x, y, z;
    cin >> x >> y >> z;
    cout << max(max(x,y), z) - min(min(x,y), z);
    return 0;
}
